src/views/Modal.ts

```diff
@@ -0,0 +1,94 @@
+import { Colors } from '../enums';
+
+class Modal {
+    element: HTMLDivElement;
+    constructor(private saveSettings: (name: string, side: Colors) => void) {
+        const modal = document.createElement('div');
+        this.element = modal;
+        this.element.classList.add('modal', 'modal--closed');
+        const potter = document.createElement('div');
+        potter.classList.add('modal__potter');
+        modal.appendChild(potter);
+        const gambit = document.createElement('div');
+        gambit.classList.add('modal__gambit');
+        modal.appendChild(gambit);
+        this.addModalForm();
+        this.openModal();
+        document.querySelector('#app')?.appendChild(modal);
```

Comment:
lepiej wyglada jak konstruktor jest krótki i ewentualnie wywołuje potrzebne metody, sugeruję wynieść to do metody typu setupModal()

Reply:
dzięki! robi się!